CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Fixed
 
+- `SingleLineInputBase`: Fixed `flex` basis setting. ([@stefaandevylder]https://github.com/stefaandevylder) in [#2014](https://github.com/teamleadercrm/ui/pull/2014)
```

Comment:
This is not really an explanation of what you're fixing. I assume you're fixing that the datepicker wasn't full width?

Reply:
Yeah but this applies for more than the datepicker I thought, or am I wrong?

Reply:
Basically when width was set to 100% the input width was 2px off. with for example setting the select input to 100%. The select however was styled correctly, but the SingleLineInputBase 2px too big. Also missing a '(' after your name. I'll fix it

Reply:
Updated the section and fixed brackets